testutil-server/src/test/java/io/spine/server/model/ModelTestsShould.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2017, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.server.model;
+
+import com.google.protobuf.Empty;
+import com.google.protobuf.Timestamp;
+import io.spine.server.command.Assign;
+import io.spine.server.command.CommandHandler;
+import io.spine.server.event.EventBus;
+import io.spine.test.Tests;
+import org.junit.Test;
+
+/**
+ * @author Alexander Yevsyukov
+ */
+public class ModelTestsShould {
+
+    @Test
+    public void have_utility_ctor() {
+        Tests.assertHasPrivateParameterlessCtor(ModelTests.class);
+    }
+
+    @Test
+    public void clear_the_model() {
+        final Model model = Model.getInstance();
+
+        // This adds new class to the model.
+        model.asCommandHandlerClass(TestCommandHandler.class);
+
+        ModelTests.clearModel();
+
+        // This should pass as we cleared the model.
```

Comment:
Not sure what you mean. `asCommandHandlerClass(..)` does not seem to throw anything in case of duplicate.